src/app/rundown-view/components/rundown-subheader/rundown-subheader.component.ts

```diff
@@ -0,0 +1,47 @@
+import { ChangeDetectionStrategy, Component, Input, OnChanges, SimpleChange, SimpleChanges } from '@angular/core'
+import { Rundown } from '../../../core/models/rundown'
+import { Piece } from '../../../core/models/piece'
+import { Tv2PieceMetadata } from '../../../core/models/tv2-piece'
+import { Tv2PieceType } from '../../../core/enums/tv2-piece-type'
+import { Part } from '../../../core/models/part'
+
+@Component({
+  selector: 'sofie-rundown-subheader',
+  templateUrl: './rundown-subheader.component.html',
+  styleUrls: ['./rundown-subheader.component.scss'],
+  changeDetection: ChangeDetectionStrategy.OnPush,
+})
+export class RundownSubheaderComponent implements OnChanges {
+  @Input()
+  public rundown: Rundown
+
+  public endWords?: string
+
+  public ngOnChanges(changes: SimpleChanges): void {
+    const rundownChange: SimpleChange = changes['rundown']
+    if (rundownChange) {
+      this.setEndWords()
+    }
```

Comment:
Updating type for `rundownChange` to include the possibility of it being undefined would help explain why there is an undefiend check.
At the moment you are checking that a value is not undefined, which doesn't appear from the type to even have the possibility.

Reply:
Nice catch – I have updated it.